pytest_splinter/plugin.py

```diff
@@ -380,37 +381,37 @@ def _take_screenshot(
 
     classname = '.'.join(names[:-1])
     screenshot_dir = os.path.join(splinter_screenshot_dir, classname)
-    screenshot_file_name_format = '{0}.{{format}}'.format(
-        '{0}-{1}'.format(names[-1][:128 - len(fixture_name) - 5], fixture_name).replace(os.path.sep, '-')
-    )
-    screenshot_file_name = screenshot_file_name_format.format(format='png')
-    screenshot_html_file_name = screenshot_file_name_format.format(format='html')
+
     if not slaveoutput:
         if not os.path.exists(screenshot_dir):
             os.makedirs(screenshot_dir)
     else:
         screenshot_dir = session_tmpdir.ensure('screenshots', dir=True).strpath
-    screenshot_png_path = os.path.join(screenshot_dir, screenshot_file_name)
-    screenshot_html_path = os.path.join(screenshot_dir, screenshot_html_file_name)
-    LOGGER.info('Saving screenshot to %s', screenshot_dir)
+
+    screenshot_base_file_name = '{0}-{1}-'.format(names[-1][:128 - len(fixture_name) - 5], fixture_name).replace(os.path.sep, '-')
+    screenshot_path = os.path.join(screenshot_dir, screenshot_base_file_name)
+
+    LOGGER.info('Saving screenshot to {}'.format(screenshot_dir))
+
     try:
-        splinter_screenshot_getter_html(browser_instance, screenshot_html_path)
-        splinter_screenshot_getter_png(browser_instance, screenshot_png_path)
+        html_snapshot_path = splinter_screenshot_getter_html(browser_instance, screenshot_path)
```

Comment:
does it make sense to have snapshot vs screenshot ambiguity?
let's not change screenshot_png_path and screenshot_html_path naming